### ISSUE TITLE
Continuous Capability?

### ISSUE THREAD

**Issue by aldorr** [dirkgroenen/simple-jQuery-slider#21]
Hi Dirk,
I was trying to figure out a way to make the animated transitions not stop on each slide, but to just slide through each image continuously.

- If I set the animateDuration to 4sec and the interval to 4sec (and animationEasing to linear?)there's a pause.
- If I set the animateDuration to 4sec and the interval to 0sec everything jitters.

Is there a practical way to do this?
Thanks for any pointers!

**Comment by dirkgroenen** [comment]
Have you tried invoking the `nextSlide()` slide on `triggerSlideEnd` in combination with the linear easing? I'm not sure if it's actually going to work, but maybe it's worth trying. 

Honestly you don't need jQuery nor Javascript to achieve this. If you put your items in an overflowing container you can actually achieve the affect by applying an CSS animation on it.

**Comment by aldorr** [comment]
Thanks for responding. I tried that as well. Unfortunately it doesn't work.

Of course, just to slide from left to right or right to left with CSS is trivial.
But I liked that I could easily pause on hover, make regions clickable – to manually slide and slide infinitesimally. :)
Now I've switched over to writing my own script & styles. Not impossible, but more time consuming... and now my code will only include the absolute necessary, which is also a good thing.
